test/unit/routes/helpers/test-get-claim-expense-non-persisted-details.js

```diff
@@ -0,0 +1,53 @@
+const expect = require('chai').expect
+
+const getNonPersistedExpenseData = require('../../../../app/routes/helpers/get-claim-expense-non-persisted-details')
+
+describe('routes/helpers/get-claim-expense-non-persisted-details', function () {
+  it('should return an array of claim expenses', function (done) {
```

Comment:
test should have more cases describing what it does:
- `should return an array of claim expenses`
- `should set status`
- `should set ApprovedCost`
- `should set Error`